wp-content/themes/core/routes/unsupported_browser/unsupported_browser.php

```diff
@@ -18,7 +18,10 @@
 
 	<?php echo $c->get_styles(); ?>
 
-	<link rel="shortcut icon" href="<?php echo esc_url( trailingslashit( get_template_directory_uri() ) . 'assets/img/theme/branding-assets/favicon.ico' ); ?>">
+	<?php if ( $c->get_favicon_url() ) { ?>
+		<link rel="shortcut icon" href="<?php echo $c->get_favicon_url(); ?>">
```

Comment:
```suggestion
		<link rel="shortcut icon" href="<?php echo esc_url( $c->get_favicon_url() ); ?>">
```
We should escape as late as possible, so in the view when we can :smile: